src/Models/Companies.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Sdk\Models;
+
+use Kanvas\Sdk\Companies as CompaniesResource;
+
+/**
+ * Users Class.
+ */
+class Companies extends BaseModel
+{
+    /**
```

Comment:
@rwhite27 if you have resource function similar to the one in phalcon setSource, just set the function and you wont have to redefine the find and findfirst methods